package.json

```diff
@@ -59,13 +57,11 @@
     "postcss-safe-parser": "3.0.1",
```

Comment:
Same question for postcss-safe-parser

Reply:
Run ‘yarn why’. Is it a peer dependency?

Reply:
```
$ yarn why postcss-safe-parser                                                                                                                                                                                                            
 🤔  Why do we have the module "postcss-safe-parser"...?
=> Found "postcss-safe-parser@3.0.1"
info Has been hoisted to "postcss-safe-parser"
info Reasons this module exists
   - Specified in "devDependencies"
   - Hoisted from "stylelint#postcss-safe-parser"
```